container/container_test.go

```diff
@@ -510,6 +510,25 @@ func TestNew(t *testing.T) {
 			},
 			wantContainerErr: true,
 		},
+		{
+			desc:     "fails on SplitFixed less than -1",
+			termSize: image.Point{10, 20},
+			container: func(ft *faketerm.Terminal) (*Container, error) {
+				return New(
+					ft,
+					SplitHorizontal(
+						Top(
+							Border(linestyle.Light),
+						),
+						Bottom(
+							Border(linestyle.Light),
+						),
+						SplitPercent(-4),
```

Comment:
Did we want to use SplitFixed() here?

(optional) As for the chosen value, it is better to choose the "edge value", i.e. the first one that would be incorrect (-2). If we choose -4, we aren't really covering -3 and -2 with this test.

Reply:
Oops, you're right, should be SplitFixed here.
-2 is a better value too, agreed, will fix.